src/main/scala/org/clulab/wm/discovery/ConceptDiscovery.scala

```diff
@@ -22,18 +25,20 @@ case class RankedConcept(concept: Concept, saliency: Double)
 
 class ConceptDiscovery {
 
-  def discoverConcepts(documents: Seq[Path]): Set[Concept] = {
+  def discoverConcepts(cdrs: Seq[CdrDocument], sentenceThreshold: Option[Double]): Set[Concept] = {
     Utils.initializeDyNet()
     val tagSet = new EnglishTagSet()
     val Config = EidosSystem.defaultConfig
     val stopwordManager = StopwordManager.fromConfig(Config, tagSet)
     val config = ConfigFactory.load("reference")
     val entityFinder = RuleBasedEntityFinder.fromConfig(config, tagSet, stopwordManager)
     val processor = new CluCoreProcessor()
+    val textListAll = new ArrayBuffer[String]()
+    val sentenceListAll = new ArrayBuffer[String]()
     val conceptLocations = mutable.Map.empty[String, Set[String]].withDefaultValue(Set.empty)
-    for (path <- documents) {
-      val text = FileUtils.getTextFromFile(path.toString)
-      val document = processor.annotate(text)
+    for ((cdr, idx) <- cdrs.zipWithIndex) {
```

Comment:
Is there a reason there's an `idx` and `zipWithIndex` here? Seems unnecessary.

Reply:
idk, it was there when i started and idk what it was/is to be used for